proto/proto/store/ntx_builder.proto

```diff
@@ -51,15 +54,35 @@ message MaybeAccountDetails {
 message UnconsumedNetworkNotesRequest {
     // An opaque token used to paginate through the notes.
     //
-    // This should be null on the first call, and set to the response
-    // token until the response token is null, at which point all data
-    // has been fetched.
+    // This should be null on the first call, and set to the response  token until the response token
+    // is null, at which point all data has been fetched.
     optional uint64 page_token = 1;
 
     // Number of notes to retrieve per page.
     uint64 page_size = 2;
 }
 
+// Returns a paginated list of network notes for an account which are unconsumed by a specified
+// block number.
+message UnconsumedNetworkNotesForAccountRequest {
+    // This should be null on the first call, and set to the response token until the response token
+    // is null, at which point all data has been fetched.
+    //
+    // Note that this token is only valid if used with the same parameters.
+     optional uint64 page_token = 1;
+
+     // Number of notes to retrieve per page.
+     uint64 page_size = 2;
+
+     // The network account ID prefix to filter notes by.
+     uint32 network_account_id_prefix = 3;
+
+     // The block number (inclusive) that the notes must be unconsumed by.
+     //
+     // Notes that are consumed after this block are included in the result.
+     fixed32 unconsumed_by_block_num = 4;
```

Comment:
I'm not sure the `unconsumed_by_block_num` name is appropriate here. If I'm understanding correctly, this block number is used for 2 things:

1. Filter out any note created before this block.
2. Filter out any note consumed after this block.

If we keep both criteria, then we should probably change the name to something else (e.g., just `block_num` or `latest_block_num` as used in other places).

For the comment, we could say something like this:
```proto
 // The block number to filter the returned notes by.
 //
 // Notes that are created or consumed after this block are excluded from the result.
```